### PR TITLE
Repaint README

### PR DESCRIPTION
**Pull request by szepeviktor**
I am old 👴🏻 
For me readme sections are
- Requirements
- (Build &) Installation
- Configuration
- Usage

But I am young: emoji added.

Preview: https://github.com/szepeviktor/emilpro/tree/patch-2#readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,5 +63,14 @@ cmake -B build -GNinja -DCMAKE_PREFIX_PATH="`pwd`build/build/Release/generators/
...
+
+_Do we need it?_
+
+## Usage
+
+_How to launch EmilPRO?_
+
+## Support
+
+Simon Kagstrom \<simon.kagstrom@gmail.com\>
```
Comment (szepeviktor) [inline README.md:76]: fix Markdown (`<  >`)

File: README.md [hunk 2]
```diff
@@ -31,7 +32,8 @@ Still TODO, but more or less like Ubuntu.
 brew install binutils qt6 conan cmake ninja
 ```
 
-## Build
+## Installation
```
Comment (szepeviktor) [inline README.md:35]: Will `ninja -C build` actually copy the binary to `/usr/`?

**Review by SimonKagstrom — APPROVED** [review]

**Comment by SimonKagstrom** [comment]
Sounds good!

I added some comments to your PR, but thanks a lot for your work!

**Comment by szepeviktor** [comment]
Is it all right now?

**Comment by SimonKagstrom** [comment]
Great, merged!

**Comment by szepeviktor** [comment]
Was joyful to make.

**Pull request merged**